Registry/Registry.EM_COMMON

```diff
@@ -488,11 +488,11 @@ state   real    dfi_qh         ikjftb   dfi_moist       1         -     \
    rusdf=(bdy_interp:dt)  "DFI_QHAIL"        "Hail mixing ratio"             "kg kg-1"
 state    real   qvold            ikj    misc        1         -     rdu      "QVOLD"          "Water vapor mixing ratio, old time step"      "kg kg-1"
 state    real   rimi             ikj    misc        1         -     irh      "RIMI"           "riming intensity"  "fraction"
-state    real   qnwfa2d          ij     misc        1         -    i014{17}rhdu "QNWFA2D"       "Surface aerosol number conc emission"  "kg-1 s-1"
-state    real   qnifa2d          ij     misc        1         -    i014{17}rhdu "QNIFA2D"       "Surface dust number conc emission"  "kg-1 s-1"
-state    real   qnbca2d          ij     misc        1         -    i014{17}rhdu "QNBCA2D"       "Surface black carbon number conc emission"  "kg-1 s-1"
-state    real   qnocbb2d         ij     misc        1         -    i014{17}rhdu "QNOCBB2D"      "Surface organic carbon biomass burning number conc emission"  "kg-1 s-1"
-state    real   qnbcbb2d         ij     misc        1         -    i014{17}rhdu "QNBCBB2D"      "Surface black carbon biomass burning number conc emission"  "kg-1 s-1"
+state    real   qnwfa2d          ij     misc        1         -     i01{17}rhdu "QNWFA2D"     "Surface aerosol number conc emission"  "kg-1 s-1"
+state    real   qnifa2d          ij     misc        1         -     i01{17}rhdu "QNIFA2D"     "Surface dust number conc emission"  "kg-1 s-1"
+state    real   qnbca2d          ij     misc        1         -     i01{17}rhdu "QNBCA2D"     "Surface black carbon number conc emission"  "kg-1 s-1"
+state    real   qnocbb2d         ij     misc        1         -     i01{17}rhdu "QNOCBB2D"    "Surface organic carbon biomass burning number conc emission"  "kg-1 s-1"
+state    real   qnbcbb2d         ij     misc        1         -     i01{17}rhdu "QNBCBB2D"    "Surface black carbon biomass burning number conc emission"  "kg-1 s-1"
```

Comment:
I think the 4 index here that was removed is needed for these to appear in the lower boundary file for time dependence to work.

Reply:
@dudhia I coded up these aerosol mods to read from a wrfqnainp file (stream 17) and be separate from wrflowinp (stream 4). I suppose the 2D aerosol fields could be read in thru wrflowinp, but I thought it was more explicit to handle them separately. Please let me know your preference, I should be able to change that pretty easily. Thanks

Reply:
@dudhia There may be advantages to separate aerosol input from other input already in wrflowinp file.